docs/reference-guides/interactivity-api/iapi-about.md

```diff
@@ -0,0 +1,191 @@
+```
+
+As you can see, directives like `data-wp-on--click` or `data-wp-show` are added as custom HTML attributes. WordPress can process this HTML on the server, handling the directives’ logic and creating the appropriate markup.
+
+### Backward compatible
+
+As the Interactivity API works perfectly with server-side rendering, you can use all the WordPress APIs, including:
+
+- **WordPress filters and actions**: You can keep using WordPress hooks to modify the HTML or even to modify directives. Additionally, existing hooks will keep working as expected.
+- **Core Translation API**: e.g. `__()` and `_e()`. You can use it to translate the text in the HTML (as you normally would) and even use those APIs on the server side of your directives. 
+
+### Optional and gradual adoption
+
+The Interactivity API pipeline promotes **progressive enhancement** by building on top of WordPress’s solid foundation and patterns. It was carefully designed not to force any use cases to pay for the costs of other use cases.
+
+For example, blocks with directives can coexist with other (interactive or non-interactive) blocks. This means that if there are other blocks on the page using other frameworks like jQuery, everything will work as expected.
+
+### Declarative and reactive
+
+The Interactivity API follows an approach similar to other popular JS frameworks by separating state, actions, and callbacks and defining them declaratively. Why declaratively?
+
+Declarative code describes **what** a program should do, while imperative code describes **how** the program should do it. Using a declarative approach, the UI automatically updates in response to changes in the underlying data. With an imperative approach, you must manually update the UI whenever the data changes. Compare the two code examples:
+
+_Imperative code_
+
+```html
+<button id="toggle-button">Toggle Element</button>
+<p>This element is now visible!</p>
+<script>
+  const button = document.getElementById("toggle-button");
+ 
+  button.addEventListener("click", () => {
+    const element = document.getElementById("element");
+    if(element) {
+      element.remove();
+    } else {
+      const newElement = document.createElement("p");
+      newElement.textContent = "This element is visible";
+      document.body.appendChild(newElement);
+    }
+});
+</script>
+```
+
+_Declarative code_
+
+This is the same use case shared above but serves as an example of declarative code using this new system. The JavaScript logic is defined in the `view.js` file of the block, and add the directives to the markup in the `render.php`.
+
+```js
+// view.js file
+ 
+import { store, getContext } from "@wordpress/interactivity";
+ 
+store( 'wpmovies', {
+  actions: {
+    toggle: () => {
+      const context = getContext();
+      context.isOpen = !context.isOpen;
+    },
+  },
+});
+```
+
+```php
+<!-- Render.php file -->
+ 
+<div
+  data-wp-interactive='{ "namespace": "wpmovies" }'
+  data-wp-context="{ 'isOpen': true }"
```

Comment:
Same - string-only namespace and use the `wp_interactivity_data_wp_context` function.